api_app/api/routes/airlock.py

```diff
@@ -30,6 +30,8 @@
 # airlock
 @airlock_workspace_router.post("/workspaces/{workspace_id}/requests", status_code=status.HTTP_201_CREATED, response_model=AirlockRequestInResponse, name=strings.API_CREATE_AIRLOCK_REQUEST, dependencies=[Depends(get_current_workspace_owner_or_researcher_user), Depends(get_workspace_by_id_from_path)])
 async def create_draft_request(airlock_request_input: AirlockRequestInCreate, user=Depends(get_current_workspace_owner_or_researcher_user), airlock_request_repo=Depends(get_repository(AirlockRequestRepository)), workspace=Depends(get_deployed_workspace_by_id_from_path)) -> AirlockRequestInResponse:
+    if workspace.properties.get("enable_airlock") is False:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=strings.AIRLOCK_NOT_ENABLED_IN_WORKSPACE)
```

Comment:
maybe 405? because the resource does exist, it is just that you can't create requests underneath it

Reply:
yes it makes more sense... changed